### PR TITLE
FIX: script and cron action yml

### LINKED ISSUE #773: CICD:  untaged docker image cleaner
### 🚀 Feature Request

We should delete untaged docker images because they use extra space.

### Proposal

- [ ] Github workflow script on cron.
     - [Github docker registry api reference](https://docs.github.com/en/rest/packages). You can use `GITHUB_TOKEN` to call api
     - Example: `curl -H "Authorization: token $github_token" -X GET "https://api.github.com/orgs/tinkoff-ai/packages?package_type=container"`
- [ ] Leave last 20 untaged images and remove other.
- [ ] Check if taged images are not at risk

### Test cases

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by martins0n**
## Before submitting (must do checklist)

- [x] Did you read the [contribution guide](https://github.com/tinkoff-ai/etna/blob/master/CONTRIBUTING.md)?
- [x] Did you update the docs? We use Numpy format for all the methods and classes.
- [x] Did you write any new necessary tests?
- [x] Did you update the [CHANGELOG](https://github.com/tinkoff-ai/etna/blob/master/CHANGELOG.md)?

## Closing issues

closes #773

### CODE REVIEW

File: scripts/remove_old_images.py [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            image_versions += parsed_result
+            page += 1
+    # filter out tagged images
+    image_versions = [image for image in image_versions if len(image["metadata"]["container"]["tags"]) == 0]
+    return image_versions
+
+
+def get_list_to_remove(leave_last_n_images: int, image_versions: list):
+    image_versions = sorted(
+        image_versions, key=lambda x: datetime.datetime.strptime(x["created_at"], "%Y-%m-%dT%H:%M:%SZ")
```
Comment (Mr-Geekman) [inline scripts/remove_old_images.py:37]: Isn't it iso-format? Probably we can use [datetime.datetime.fromisoformat](https://docs.python.org/3/library/datetime.html#datetime.datetime.fromisoformat).

File: scripts/remove_old_images.py [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+    )
+    return image_versions[:-leave_last_n_images]
+
+
+def remove_images(image_versions: list):
+    for image in image_versions:
+        print(f"Removing {image['url']}")
+        subprocess.run(
```
Comment (Mr-Geekman) [inline scripts/remove_old_images.py:45]: Why do we need such a strange way of running this code?

File: scripts/remove_old_images.py (hunk 1 above)
Comment (martins0n) [reply scripts/remove_old_images.py:37]: Z at the end

File: scripts/remove_old_images.py (hunk 2 above)
Comment (martins0n) [reply scripts/remove_old_images.py:45]: For some reason non-get empty requests return 422 error.
This is workaround.

https://github.com/cli/cli/issues/3937

**Review by Mr-Geekman — APPROVED** [review]

**Pull request merged**